.ai/README.md

```
@@ -85,6 +85,13 @@ Documentation about documentation:
 - Pest testing patterns
 - Laravel conventions
 
+### Docker Compose Extensions
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)